### PR TITLE
expo upgrade

### PR DESCRIPTION
**Pull request by sallesma**
Before sending your pull-request 💌 make sure :

- [x] The tests are up-to-date and your code is tested

Closes https://github.com/disc-in/UltimateApp/issues/284

### CODE REVIEW

File: src/Components/EssentialPage.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import { View, StyleSheet, Text, TouchableOpacity, FlatList, ImageBackground } from 'react-native';
-import { Picker } from '@react-native-community/picker';
+import { Picker } from '@react-native-picker/picker';
```
Comment (sallesma) [inline src/Components/EssentialPage.js:3]: There is this error while running the tests

```
    SyntaxError: Unexpected token 'export'

    > 3 | import { Picker } from '@react-native-picker/picker';
        | ^

```

Could be related to https://github.com/react-native-picker/picker/issues/98

File: package-lock.json [hunk 2]
```diff
@@ -1376,46 +1296,20 @@
...
-            "signal-exit": "^3.0.2"
-          }
         }
       }
     },
     "@expo/vector-icons": {
-      "version": "10.2.1",
-      "resolved": "https://registry.npmjs.org/@expo/vector-icons/-/vector-icons-10.2.1.tgz",
-      "integrity": "sha512-clYQZFLeU2y23n03hXg18EEsZS5c73sJJnfderztfSAqkUXkfUtv07fwuprYwbHIvgFkw6L7R6xJOCVYtS85iQ==",
+      "version": "12.0.2",
```
Comment (sallesma) [inline package-lock.json:1303]: Les icônes de la home page changent car Ionicons a été mis à jour

image

Comment (sallesma) [reply package-lock.json:1303]: Du coup je propose ça

image

Comment (guillaumemoby) [reply package-lock.json:1347]: Finalement c'est bon pour moi

File: src/Components/EssentialPage.js (hunk 1 above)
Comment (sallesma) [reply src/Components/EssentialPage.js:3]: Came from transformIgnorePattern not being defined: https://docs.expo.io/guides/testing-with-jest/#jest-configuration

**Review by guillaumemoby — APPROVED** [review]

**Pull request merged**